src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPLINPACK.json

```diff
@@ -6,7 +6,7 @@
     "SupportedOperatingSystems": "Ubuntu"
   },
   "Parameters": {
-    "Username": "",
+    "Username": "testuser",
```

Comment:
Hey @nmalkapuram I made the change that the username will default to the Environment.UserName if the profile doesn't specify one, is that working? Do you have to make this change?

Reply:
It wasn't working. It's taking as null value, if isn't supplied from command line.